crux-llvm/crux-llvm.cabal

```diff
@@ -131,9 +132,16 @@ executable crux-llvm-svcomp
     async >= 2.2,
     ansi-terminal,
     crux-llvm,
+    lumberjack,
     time,
     unix
 
+  other-modules:
+    Paths_crux_llvm
+
+  autogen-modules:
+    Paths_crux_llvm
```

Comment:
Probably because the sv_comp/Main.hs includes this directly?  Under GHC 8.10 and cabal-install version 3.2.0.0, I get the warning about

```
<no location info>: warning: [-Wmissing-home-modules]
    These modules are needed for compilation but not listed in your cabal file's other-modules:
        Paths_crux_llvm
```

Reply:
Now that you mention it, I'm able to reproduce this as well. How baffling. There's something a bit strange about this, but in the meantime, I support specifying `Paths_crux_llvm` here (perhaps redundantly) to avoid warnings.